DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+    "![](https://i.imgur.com/z1FkMNB.png)  |  ![](https://i.imgur.com/qIPZvc2.png)\n",
+    "\n",
+    "We see that even in a case where you're not trying to put your bias into the search, that your results can be skewed. This can be resolved by looking at aggregates of data instead of single-sources. This makes way for \"pop science\" where a study finds that X causes/cures Y, where both X and Y are common conditions. We see more how these unfortunate statistical anomalies become daytime news stories here.\n",
+    "\n",
+    "\n",
+    "#### Data Dredging\n",
+    "\n",
+    "Data dredging is the effect when intentional or not, you have too many variables or opportunities for correlation for a single dataset. There is a famous example of this, when the website FiveThirtyEight reported on the specific field of nutrition, as the data available has issues already. They noted the specific issues of how they generate \"huge data sets with many, many variables,\" which are the most susceptible data dredging. [The article](https://fivethirtyeight.com/features/you-cant-trust-what-you-read-about-nutrition/) finds a few particularly remarkable relationships, where they found the following links from the respondent data:\n",
+    "\n",
+    "![](https://i.imgur.com/CoKDCuc.png)\n",
+    "\n",
+    "You'll (hopefully) not be surprised to know that there is no correlation in real life between cabbage and innie bellybuttons or tomatoes and Judaism. This is a function, as the article explains, of over 27,000 regressions being used over less than 60 complete responses, where they expect a 5% false-positive rate. Why 5%? well, that goes into the next topic, p-hacking.\n",
+    "\n",
+    "\n",
+    "#### P-Hacking and Cherry Picking\n",
+    "\n",
+    "You might have noticed the far-right column in the above picture has a \"p-value\" label. This is the value that statisticians use to measure the risk of a false positive. In general, studies with a p-value of less than 0.05, or 5%, can be published. It is a hard burden to prove in some situations, but in some situations it is absurdly easy. Linked in the above article is an [interactive piece](https://projects.fivethirtyeight.com/p-hacking/) by FiveThirtyEight that allows you to use p-hacking in action. This one comes with a bonus of letting you lean into confirmation bias as well, as it contains economic and political data from the last ~70 years. You get to cherry-pick the types of political data to use and the ways to measure a \"good\" economy, and the right panel allows you to see how significant your results are. It should go without saying that if you cherry-pick enough, you can prove any conclusion you want to.\n",
+    "\n",
+    "This phenomenon is very possible in the transfer of information at any stage from data sources to scientists to reporters to consumers, again intentional or unintentional. We see this taken to a comedic level with XKCD's exploration of jelly beans:\n",
```

Comment:
Would reword to: This phenomenon is very possible during the transfer of information at any stage — whether from  from data sources to scientists to reporters to consumers — and it can be intentional or unintentional.